eslint.config.mjs

```diff
@@ -8,7 +8,6 @@ const config = createConfig([
   {
     ignores: [
       '**/assembly',
-      '**/build',
```

Comment:
This was ignoring `src/commands/build` in the CLI, which contained some ESLint errors. I don't think we use "build" for anything else.

Reply:
https://github.com/MetaMask/snaps/blob/main/packages/examples/packages/wasm/src/index.ts#L8

Reply:
They'll only show up once you've built the packages. But we should prevent that